Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Stage 1: Use a Golang image to get protoc-gen-go
+FROM golang:latest AS go_builder
```

Comment:
let's use the same specific golang version here so we don't drift at all

Reply:
Good point, done!